pkgs/development/compilers/llvm/11/default.nix

```diff
@@ -271,23 +275,29 @@ let
 
     libcxx = callPackage ./libcxx {
       inherit llvm_meta;
-      stdenv = if (stdenv.hostPlatform.useLLVM or false) || (stdenv.hostPlatform.isDarwin && stdenv.hostPlatform.isAarch64)
-               then overrideCC stdenv buildLlvmTools.clangNoLibcxx
-               else stdenv;
+      stdenv = if stdenv.hostPlatform.isDarwin && (stdenv.buildPlatform != stdenv.hostPlatform)
+        then overrideCC darwin.stdenvNoCF buildLlvmTools.clangNoLibcxx
+        else if stdenv.hostPlatform.useLLVM or false || stdenv.isDarwin
```

Comment:
I made the change to match how the newer LLVM derivations are written. It may not strictly need it, but it’s easier when both platforms follow the same code paths.

Reply:
Minor issue anyway, fine.